resources/views/profile/edit.blade.php

```diff
@@ -175,6 +176,8 @@
                 profileUpdate() {
                     this.resetErrors();
                     if (!this.validatePassword()) return false;
+                    if (@json($enabled2FA) && typeof this.formData.preferences_2fa != "undefined" &&
+                        this.formData.preferences_2fa.length != null && this.formData.preferences_2fa.length < 1) return false;
```

Comment:
Is this right? I think it should be:

```suggestion
                        this.formData.preferences_2fa != null && this.formData.preferences_2fa.length < 1) return false;
```

Reply:
Split this 127 characters long line (which is greater than 120 authorized).

![image](https://github.com/ProcessMaker/processmaker/assets/8028650/9d3cd929-f497-4937-a601-11a4cb60d32c)